src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java

```diff
@@ -158,9 +162,75 @@ private static boolean isFollowedByWhitespace(DetailAST targetAST, int... line)
 
             followedByWhitespace = codePoint == ';'
                 || codePoint == ')'
+                || codePoint == ','
```

Comment:
One problem is that this change has an impact on other tokens (not only on `TYPE`), because the method is shared.